app/models/manageiq/providers/ibm_power_hmc/infra_manager.rb

```diff
@@ -20,6 +20,8 @@ class ManageIQ::Providers::IbmPowerHmc::InfraManager < ManageIQ::Providers::Infr
   supports :provisioning
 
   has_many :hosts_advanced_settings, :through => :hosts, :source => :advanced_settings
+  has_many :miq_scsi_targets, -> { distinct }, :through => :guest_devices
```

Comment:
Hi @agrare, do you think we have to create an empty class for MiqScsiTargets and MiqScsiLuns inside this repository ? Is it something conventional ?

Reply:
Hey @CallisteH no you don't need to, those tables don't have STI so they can't be subclassed